build.gradle

```diff
@@ -30,6 +30,9 @@ subprojects {
 
     dependencies {
         testCompile "org.testng:testng:6.14.3"
+        testCompile "com.google.jimfs:jimfs:1.1"
+
+        compile 'commons-io:commons-io:2.5'
```

Comment:
I forgot to mention: is this needed in this PR?

Reply:
Good catch - its left over from previous changes and can be removed.